src/client/components/EmployersQuestions/index.js

```diff
@@ -47,15 +47,28 @@ function EmployerQuestions(props) {
     dateTokens.length === 3 ? dateTokens[1] : ""
   );
 
-  function renderTextInput(name, className) {
+  /**
+   * Create a text input element.
+   * @param {string} name The name for the employer data field.
+   * @param {Object?} options Optional controls:
+   *     className: The classname for applying styles.
```

Comment:
Ugh, these alerts are so annoying! Do you mind adding that option to the eslint file?

Reply:
Oops, fixed.